JuniorStart/Services/UserService.cs

```diff
@@ -10,17 +12,20 @@ namespace JuniorStart.Services
     public class UserService : IUserService
     {
         private readonly ApplicationContext _context;
+        private readonly IModelFactory _modelFactory;
 
-        public UserService(ApplicationContext context)
+        public UserService(ApplicationContext context, IModelFactory modelFactory)
         {
             _context = context;
+            _modelFactory = modelFactory;
         }
 
-        public bool Create(User user)
+        public bool Create(UserViewModel userViewModel)
         {
-            if (_context.Users.FirstOrDefault(x => x.Login == user.Login) != null)
-                throw new Exception("Username \"" + user.Login + "\" is already taken");
-
+            if (_context.Users.FirstOrDefault(x => x.Login == userViewModel.Login) != null)
```

Comment:
osobiście wole dorzucać klamry nawet jak if jest jedno linijkowy
ostatecznie jest to bardziej czytelne